contracts/actions/aaveV3/AaveV3Supply.sol

```diff
@@ -134,25 +122,25 @@ contract AaveV3Supply is ActionBase, AaveV3Helper {
 
         lendingPool.supply(tokenAddr, _amount, _onBehalf, AAVE_REFERRAL_CODE);
 
-        if (_enableAsColl) {
-            lendingPool.setUserUseReserveAsCollateral(tokenAddr, true);
-        } else {
-            lendingPool.setUserUseReserveAsCollateral(tokenAddr, false);
-        }
-
         bytes memory logData = abi.encode(
             _market,
             tokenAddr,
             _amount,
             _from,
             _onBehalf,
-            _enableAsColl
+            true
```

Comment:
would remove it here to as it's unnecessary (we could potentially check aave pool to see if asset that we supplied is now enabled as collateral and then log that, as in some cases after .supply() the asset won't be enabled as collateral)